.storybook/Story.scss

```diff
@@ -460,6 +460,10 @@ body.sb-show-main {
       background-color: transparent;
     }
   }
+
+  pre + pre {
```

Comment:
This reduces the vertical gap between adjacent code samples, feels little tighter and more legible